### PR TITLE
refactor codes in volume iscsi to improve readability

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

**What this PR does / why we need it**:
This PR refactors some codes in pkg/volume/iscsi. What is specific, this PR takes advantage of return fast to make codes indent less. As a result the readability of codes will improve a little bit.

What I did:
1.  refactor codes in volume iscsi to improve readability.
2.  change a keyword of `delete` into `deleteArgs` to reduce ambiguousness.
3.  make some variables camel case. 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
NONE

**Special notes for your reviewer**:
NONE

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by allencloud** [comment]
ping @jsafrane @saad-ali @humblec @mtanino 
Could you help to review this? And feel free to tell me if I missed anything. 🍺

**Comment by jsafrane** [comment]
/approve

If I did not miss a dangling `else` somewhere (and I tried hard), the code looks semantically equivalent. I let @rootfs or @mtanino to judge if they like this new style.

File: pkg/volume/iscsi/iscsi_util.go [hunk 1]
```diff
@@ -45,41 +45,45 @@ var (
...
+
+	for _, k := range chap_sess {
+		v := b.secret[k]
+		if len(v) > 0 {
+			out, err := b.exec.Run("iscsiadm", "-m", "node", "-p", tp, "-T", b.Iqn, "-I", b.Iface, "-o", "update", "-n", k, "-v", v)
+			if err != nil {
+				return fmt.Errorf("iscsi: failed to update node session key %q with value %q error: %v", k, v, string(out))
 			}
 		}
+
```
Comment (mtanino) [inline pkg/volume/iscsi/iscsi_util.go]: @allencloud 
Please remove this unnecessary blank line.

**Comment by mtanino** [comment]
@rootfs @jsafrane @allencloud 
I also confirmed the refactored code is semantically same as previous logic.
I'm OK for this refactoring.

As I mentioned above, please remove unnecessary blank line.
Otherwise, loos good to me.

**Comment by allencloud** [comment]
Thanks for your feedback. I have updated the PR and removed the blank line. PTAL @mtanino

**Comment by mtanino** [comment]
LGTM.

**Comment by allencloud** [comment]
Thanks @mtanino 
Could you add a label `/lgtm` ?  So that I think we could put this PR into the merge queue.

File: pkg/volume/iscsi/iscsi_util.go [hunk 2]
```diff
@@ -338,84 +344,87 @@ func (util *ISCSIUtil) DetachDisk(c iscsiDiskUnmounter, mntPath string) error {
...
+		}
+	}
+
 	return nil
 }
 
 func extractTransportname(ifaceOutput string) (iscsiTransport string) {
 	re := regexp.MustCompile(`iface.transport_name = (.*)\n`)
 
 	rex_output := re.FindStringSubmatch(ifaceOutput)
```
Comment (rootfs) [inline pkg/volume/iscsi/iscsi_util.go]: nit: camel case

**Comment by rootfs** [comment]
@allencloud a small nit. 

since #50334 is ready to merge, please check if additional improvement can be made

**Comment by allencloud** [comment]
#50334 has already been merged. And I have solved the conflict which is caused by the newly merged PR.

ping @rootfs @mtanino  PTAL.

**Comment by rootfs** [comment]
thank  you @allencloud

**Pull request merged**